docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+    api-dummy-fpm:
+        build:
+            context: ./api-dummy
+        volumes:
+            - ./api-dummy:/srv/app
+        environment:
+            - APP_ENV=dev
+            - APP_SECRET=f6a87c2e031ecadc0d7eb442729435c5
+    api-dummy-nginx:
+        image: nginx:1.13
+        ports:
+            - '8081:80'
+        volumes:
+            - ./api-dummy/.docker/nginx-server.conf:/etc/nginx/conf.d/default.conf
+            - ./api-dummy/public:/srv/app/public
+        depends_on:
+            - api-dummy-fpm
+    journal-fpm:
+        build:
+            context: ./journal
+        volumes:
+            - ./journal:/srv/app
```

Comment:
for both projects only `/srv/app/src` (and other relevant subfolders) should be mounted or `var/cache/` starts showing permission problems when the container user does not match the host folder owner